src/main/res/values/strings.xml

```diff
@@ -17,25 +17,25 @@
   <string name="anti_feature_non_free_dependencies">Depends on non-free applications</string>
   <string name="anti_feature_non_free_network">Promotes non-free network services</string>
   <string name="anti_feature_tracking">Tracks or reports your activity</string>
-  <string name="anti_feature_upstream_non_free">Upstream source code is not free</string>
+  <string name="anti_feature_upstream_non_free">Upstream source code is not libre</string>
   <string name="anti_features">Anti-features</string>
   <string name="app_name">Foxy Droid</string>
   <string name="application">Application</string>
   <string name="application_not_found">Application not found</string>
-  <string name="authors_website">Author\'s website</string>
-  <string name="authors_email">Author\'s email</string>
+  <string name="authors_website">Author website</string>
+  <string name="authors_email">Author e-mail</string>
   <string name="available">Available</string>
   <string name="available_empty_summary">No applications available</string>
   <string name="bug_tracker">Bug tracker</string>
   <string name="cancel">Cancel</string>
   <string name="cant_edit_sync_description">Cannot edit repository since it\'s syncing right now.</string>
   <string name="changelog">Changelog</string>
   <string name="checking_repository">Checking repository</string>
-  <string name="compatible_with_only_format">Compatible with %s only</string>
-  <string name="confirm_action">Confirm action</string>
+  <string name="compatible_with_only_format">Only compatible with %s</string>
+  <string name="confirm_action">Confirm</string>
```

Comment:
I think "Confirm action" should be used here. It's a dialog title, and then a description follows which explains which action to confirm.